core/object/make_virtuals.py

```diff
@@ -16,14 +16,24 @@
 		}    \\
 	}\\
     if (unlikely(_get_extension() && !_gdvirtual_##m_name##_initialized)) {\\
-        /* TODO: C-style cast because GDExtensionStringNamePtr's const qualifier is broken (see https://github.com/godotengine/godot/pull/67751) */\\
-        _gdvirtual_##m_name = (_get_extension() && _get_extension()->get_virtual) ? _get_extension()->get_virtual(_get_extension()->class_userdata, (GDExtensionStringNamePtr)&_gdvirtual_##m_name##_sn) : (GDExtensionClassCallVirtual) nullptr;\\
+        _gdvirtual_##m_name = (GDExtensionClassCallVirtual) nullptr;\\
```

Comment:
We probably shouldn't keep `GDExtensionClassCallVirtual` as the type on `_gdvirtual_##m_name`, because that could be confusing. We should either change this to `void *` or a "void function pointer" like `GDExtensionInterfaceFunctionPtr` (to ensure the correct alignment on platforms where that might matter). We can then cast to `GDExtensionClassCallVirtual` before calling it.

Reply:
Changed to a void* and also renamed the variable to `_gdvirtual_##m_name##_data`

Reply:
Thanks! I don't know that the change to `_gdvirtual_##m_name##_data` is really necessary, but it's fine :-)

Reply:
Wanted to be *really* clear what it was :-)